packages/lesmis/src/views/DashboardReportTabView.js

```diff
@@ -0,0 +1,92 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ *
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { SmallAlert } from '@tupaia/ui-components';
+import { useDashboardData } from '../api/queries';
+import {
+  FetchLoader,
+  TabBar,
+  TabBarLoader,
+  FlexCenter,
+  Report,
+  TabBarTabs,
+  TabBarTab,
+  TabPanel,
+  YearSelector,
+} from '../components';
+
+const DashboardSection = styled(FlexCenter)`
+  min-height: 31rem;
+`;
+
+export const DashboardReportTabView = ({
+  entityCode,
+  selectedDashboard,
+  setSelectedDashboard,
+  selectedYear,
+  setSelectedYear,
+}) => {
+  const { data, isLoading, isError, error } = useDashboardData(entityCode);
+
+  const handleChangeDashboard = (event, newValue) => {
+    setSelectedDashboard(newValue);
+  };
+
+  return (
+    <>
+      <TabBar>
+        {isLoading ? (
+          <TabBarLoader />
+        ) : (
+          <>
+            <YearSelector selectedYear={selectedYear} setSelectedYear={setSelectedYear} />
+            <TabBarTabs
+              value={selectedDashboard}
+              onChange={handleChangeDashboard}
+              variant="scrollable"
+              scrollButtons="auto"
+            >
+              {Object.entries(data).map(([heading]) => (
+                <TabBarTab key={heading} label={heading} value={heading} />
+              ))}
+            </TabBarTabs>
+          </>
+        )}
+      </TabBar>
+      <DashboardSection>
+        <FetchLoader isLoading={isLoading} isError={isError} error={error}>
+          {data &&
+            Object.entries(data).map(([heading, dashboardGroup]) => (
+              <TabPanel key={heading} dataKey={heading} value={selectedDashboard}>
+                {Object.entries(dashboardGroup).map(([groupName, groupValue]) => {
+                  const views = groupValue.views.filter(report => report.type === 'chart');
```

Comment:
```suggestion
                  const dashboardReports = groupValue.views.filter(report => report.type === 'chart'); // TODO support other report types
```
Two points there, the rename and the todo comment